submissions/damilolaSalami/domManipulation/signUpform.js

```diff
@@ -0,0 +1,42 @@
+let submitform = document.getElementById("validateForm");
+    firstname = document.getElementById("firstname");
+    lastname = document.getElementById("lastname");
+    othernames = document.getElementById("othernames");
+    phonenumber = document.getElementById("phoneNumber");
+    email = document.getElementById("email");
+    password = document.getElementById("password");
+    confirmpassword= document.getElementById("confirmpassword");
+
+var nameRegex =  /^[a-zA-Z ]+$/;
+var pnumberRegex = /^[+]*[0-9]+$/;
+var emailRegex = /^\w+([-]?\w+)*@\w+([-]?\w+)*(\.\w{2,6})+$/;
+var passwordRegex = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[._$,/;:+=~`'"<>^#?!@$%^&*-]).{6,}$/;
+
+function formValidation (field, Regex, error_one, error_two, event) {
+    if (field.value === "") {
+        field.nextElementSibling.innerHTML = error_one;
+        event.preventDefault();
+        return false;
+    }
+
+    if (!field.value.match(Regex)) {
```

Comment:
I think you should attach this logic to the initial logic with a `else if` @queensaldam